src/main/java/eu/enmeshed/EnmeshedOnboardingService.java

```diff
@@ -216,56 +201,37 @@ public RegistrationResult checkRegistrationState(
     }
 
     Relationship relationship = relationships.get(0);
-    RelationshipChange relationshipChange = relationship.getChanges().get(0);
-
-    RelationshipChangeRequestContent relationshipChangeRequestContent =
-        relationshipChange.getRequest().getContent();
-
-    RelationshipCreationChangeRequestContent relationshipCreationChangeRequestContent =
-        (RelationshipCreationChangeRequestContent) relationshipChangeRequestContent;
-
+    RelationshipCreationContent creationContent = relationship.getCreationContent();
     Map<Class<? extends AttributeValue>, AttributeValue> attributes =
-        getSharedSimpleAttributesFromResponseItems(
-            relationshipCreationChangeRequestContent.getResponse().getItems());
+        getSharedSimpleAttributesFromResponseItems(creationContent.getResponse().getItems());
 
-    if (relationshipChange.getStatus() == RelationshipChange.Status.PENDING) {
+    if (relationship.getStatus() == RelationshipStaus.PENDING) {
```

Comment:
"RelationshipStatus" instead of "RelationshipStaus"

Reply:
Fixed.